src/aks-preview/azext_aks_preview/aks_app_commands/aksappcommands.py

```diff
@@ -0,0 +1,165 @@
+# pylint: disable=too-many-lines
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import sre_parse
+from typing import List, Optional
+import shutil
+import subprocess
+import requests
+import os
+import platform
+from pathlib import Path
+from knack.prompting import prompt_y_n
+
+
+def aks_draft_app_init(deployment_path: str,
+                       app_name: str,
+                       language: str,
+                       create_config: str,
+                       dockerfile_only: str,
+                       deployment_only: str) -> None:
+    file_path = _binary_pre_check()
+    if not file_path:
+        raise ValueError('Binary check was NOT executed successfully')
+
+    arguments = _build_arguments(app_name, language, create_config, dockerfile_only, deployment_only)
+    run_successful = _run(file_path, deployment_path, arguments)
+    if run_successful:
+        _cmd_finish()
+    else:
+        raise ValueError('\'az aks app init\' was NOT executed successfully')
+
+
+# If setup is valid this method returns the correct binary path to execute
+def _binary_pre_check() -> str:
+    print('The DraftV2 setup is in progress...')
```

Comment:
Consider using logger to output different levels of prompts.

Note that the content of print is output to stdout. With logger, the content above the warning level is output to stderr, and the content at the info level and below is not output by default.

Reply:
Hey Fuming, can you link me to an example for writing live test cases that I can follow?

Reply:
live test cases in [aks-preview](https://github.com/Azure/azure-cli-extensions/blob/main/src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py), [azure-cli/acs](https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py)